### PR TITLE
[T2A5][W10-A1]Karen Ang

### PR DESCRIPTION
**Pull request by karen**

### CODE REVIEW

**Comment by jeffryhartanto** [comment]
Please mention your Team ID, e.g. [W10-A1]. Make sure you use the correct title next time.

**Comment by karen** [comment]
Hi Jeffry, noted, thanks. I did not know my team number at the time because I was overseas and unable to attend classes.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -1182,5 +1182,19 @@ private static String removePrefixSign(String s, String sign) {
     private static ArrayList<String> splitByWhitespace(String toSplit) {
         return new ArrayList(Arrays.asList(toSplit.trim().split("\\s+")));
     }
+    
+    /**
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/AddressBook.java:1186]: Good commenting style!

**Comment by OuyangDanwen** [comment]
Your code is fine. Just remember to follow the format closely for making pull request.

**Pull request closed without merging**